fuel-vm/src/storage/blob_data.rs

```diff
@@ -0,0 +1,92 @@
+use fuel_storage::Mappable;
+use fuel_types::{
+    fmt_truncated_hex,
+    BlobId,
+};
+
+use alloc::vec::Vec;
+use derivative::Derivative;
+
+#[cfg(feature = "random")]
+use rand::{
+    distributions::{
+        Distribution,
+        Standard,
+    },
+    Rng,
+};
+
+/// The storage table for blob data bytes.
+pub struct BlobData;
```

Comment:
Unsure whether this should live here or in `fuel-core`. The pre-existing types are scattered around these two repos.